noetic/distribution.yaml

```diff
@@ -6928,6 +6928,22 @@ repositories:
       url: https://github.com/ros-visualization/visualization_tutorials.git
       version: noetic-devel
     status: maintained
+  vrpn_client_ros:
+    doc:
+      type: git
+      url: https://github.com/ros-drivers/vrpn_client_ros.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/ros-drivers-gbp/vrpn_client_ros-release.git
+      version: 0.2.2-1
+    source:
+      test_pull_requests: true
```

Comment:
@paulbovbel I'm pretty sure you did, but have you configured your repository for [pull request testing](http://roswiki.autolabor.com.cn/buildfarm(2f)Pull(20)request(20)testing.html)?